src/PHPUnit/Constraint/WebDriver.php

```diff
@@ -1,25 +1,32 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\PHPUnit\Constraint;
 
 use Codeception\Exception\ElementNotFound;
 use Codeception\Lib\Console\Message;
 use Codeception\Util\Locator;
+use Facebook\WebDriver\WebDriverBy;
+use PHPUnit\Framework\ExpectationFailedException;
 use SebastianBergmann\Comparator\ComparisonFailure;
+use function count;
+use function htmlspecialchars_decode;
+use function strpos;
 
 class WebDriver extends Page
 {
-
-    protected function matches($nodes) : bool
+    protected function matches($nodes): bool
```

Comment:
Constraint logic is complicated and requires testing, but it seems that $nodes in WebDriver constraints is an array of WebDriverElements